packages/inscription-view/src/components/browser/BrowserBody.tsx

```diff
@@ -18,7 +18,12 @@ const BrowserBody = ({ open, tabs, activeTab, onTabsChange, onApply, disableAppl
 
   return (
     <DialogPortal container={editorRef.current}>
-      <DialogContent className={`browser-dialog ${!open ? 'browser-content-exit' : ''}`}>
+      <DialogContent
+        className={`browser-dialog ${!open ? 'browser-content-exit' : ''}`}
+        onInteractOutside={e => {
+          e.preventDefault();
+        }}
```

Comment:
browser in browser problem?
I assume we wouldn't have this problem if we use the ui-components browser? 🙈

Reply:
Actually, the problem lies with the select I use in the condition builder. When I open it and then click outside the dropdown while it is still open, it closes the entire browser. I believe this happens because the dropdown content is rendered outside the dialog. I was able to fix the issue with this. And indeed, this phenomenon does not occur with the ui-components browser.